__tests__/comment.test.js

```diff
@@ -0,0 +1,101 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../API/app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  describe("Functionality", () => {
+    test("Should return an Array of comment Objects", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments.length).toEqual(11);
+          comments.forEach((comment) => {
+            expect(comment.constructor).toEqual(Object);
+          });
+        });
+    });
+    test("Comment Objects should be returned in the expected format", () => {
```

Comment:
You could condense the two tests and add in line 23 before your forEach on line 35 👍